mix.exs

```diff
@@ -31,11 +31,11 @@ defmodule EctoSoftDelete.Mixfile do
 
   defp deps do
     [
-      {:ecto_sql, "~> 3.0"},
-      {:postgrex, ">= 0.0.0", only: [:test]},
-      {:ex_doc, "~> 0.16", only: [:dev, :test]},
-      {:credo, "~> 1.0", only: [:dev, :test]},
-      {:excoveralls, "~> 0.8", only: [:dev, :test]}
+      {:ecto_sql, "~> 3.11.0"},
+      {:postgrex, "~> 0.17.0", only: [:test]},
+      {:ex_doc, "~> 0.30.6", only: [:dev, :test]},
+      {:credo, "~> 1.7", only: [:dev, :test]},
+      {:excoveralls, "~> 0.17.1", only: [:dev, :test]}
```

Comment:
It seems like based on [this comment](https://github.com/revelrylabs/ecto_soft_delete/pull/154#issuecomment-2136070905) we can leave the rest of these as-is and bump ecto_sql to the minimum release that's compatible with the new code... something like this?

```suggestion
      {:ecto_sql, "~> 3.5"},
      {:postgrex, ">= 0.0.0", only: [:test]},
      {:ex_doc, "~> 0.16", only: [:dev, :test]},
      {:credo, "~> 1.0", only: [:dev, :test]},
      {:excoveralls, "~> 0.8", only: [:dev, :test]}
```